src/AstGen.zig

```diff
@@ -1722,6 +1722,55 @@ fn structInitExpr(
         }
     }
 
+    // First: The first found field.
+    // Duplicates: A list of following duplicate field names.
+    const DupeNameRecord = struct { first: Ast.TokenIndex, duplicates: ArrayListUnmanaged(Ast.TokenIndex) };
```

Comment:
These comments IMO aren't particularly helpful and can just be deleted, but if you would prefer to keep them, please make them doc comments on the actual fields:
```suggestion
    const DupeNameRecord = struct {
        /// The first field with this name.
        first: Ast.TokenIndex,
        /// Any duplicate fields with this name.
        duplicates: ArrayListUnmanaged(Ast.TokenIndex),
    };
```